### PR TITLE
e2e conformance using containerized kubelet

### PR DESCRIPTION
**Pull request by dims**

### CODE REVIEW

**Comment by dims** [comment]
/hold

need https://github.com/kubernetes/kubernetes/pull/62057 to merge

**Comment by dims** [comment]
Also, not sure if `DOCKERIZE_KUBELET` will pass through all the way to kubetest. will have to try it out and fix things if needed

File: jobs/config.json [hunk 1]
```diff
@@ -12992,6 +12992,20 @@
...
+      "--build=bazel",
+      "--deployment=local",
+      "--ginkgo-parallel=1",
+      "--provider=local",
+      "--test_args=--ginkgo.focus=\\[Conformance\\]",
+      "--timeout=120m"
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "sig-testing"
```
Comment (kargakis) [inline jobs/config.json]: I don't think sig-testing owns containerized kubelet. Is this sig-storage and sig-node?

File: prow/config.yaml [hunk 2]
```diff
@@ -2813,6 +2813,45 @@ presubmits:
...
+    agent: kubernetes
+    branches:
+    - master
+    always_run: false
+    skip_report: false
+    max_concurrency: 8
+    context: pull-kubernetes-local-e2e-containerized
+    rerun_command: "/test pull-kubernetes-local-e2e-containerized"
+    trigger: "(?m)^/test pull-kubernetes-local-e2e-containerized,?(\\s+|$)"
+    run_if_changed: '(hyperkube|local-up-cluster)'
```
Comment (kargakis) [inline prow/config.yaml]: I can't find any of these paths in kubernetes/kubernetes. Shouldn't this be something like "pkg/kubelet"?

File: prow/config.yaml [hunk 3]
```diff
@@ -2813,6 +2813,45 @@ presubmits:
       - name: docker-graph
         emptyDir: {}
 
+  - name: pull-kubernetes-local-e2e-containerized
+    agent: kubernetes
+    branches:
+    - master
+    always_run: false
+    skip_report: false
+    max_concurrency: 8
```
Comment (kargakis) [inline prow/config.yaml]: Why is this bounded to 8?

**Comment by dims** [comment]
@kargakis the whole job is copied from a previous job. will update what you have pointed out. or even feel free to copy from this PR and take over. Just wanted to give a running start based on what we already have

File: prow/config.yaml (hunk 2 above)
Comment (dims) [reply prow/config.yaml]: Done

File: prow/config.yaml (hunk 3 above)
Comment (dims) [reply prow/config.yaml]: What would you like this to be?

**Comment by dims** [comment]
reviewed the container and the bootstrap.py, DOCKERIZE_KUBELET should pass through to kubetest. now we wait for the commit  mentioned above to merge

File: prow/config.yaml (hunk 3 above)
Comment (kargakis) [reply prow/config.yaml]: Remove altogether.

**Comment by kargakis** [comment]
> need kubernetes/kubernetes#62057 to merge

merged

File: prow/config.yaml (hunk 3 above)
Comment (dims) [reply prow/config.yaml]: Done

**Comment by dims** [comment]
this is ready now. thanks!

File: prow/config.yaml [hunk 4]
```diff
@@ -4711,6 +4749,53 @@ presubmits:
...
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: security
+    context: pull-security-kubernetes-local-e2e-containerized
+    labels:
+      preset-k8s-ssh: "true"
+      preset-service-account: "true"
+    max_concurrency: 8
```
Comment (kargakis) [inline prow/config.yaml]: Needs to be removed from here too I guess

Comment (dims) [reply prow/config.yaml]: 👍  doing it now

Comment (dims) [reply prow/config.yaml]: looks like hack/update-config.sh defaults to `max_concurrency: 0`

**Comment by dims** [comment]
@kargakis fixed. please take a look

**Comment by kargakis** [comment]
/lgtm

needs a sign-off from @BenTheElder / @krzyzacy

**Comment by dims** [comment]
@krzyzacy I want to make sure it works before we can turn it into periodic job

**Comment by kargakis** [comment]
@dims it seems this job is broken, can you triage the failure?

https://prow.k8s.io/?job=pull-kubernetes-local-e2e-containerized

**Comment by dims** [comment]
@kargakis yep, working on it in https://github.com/kubernetes/kubernetes/pull/62508

**Pull request merged**